web/lattice_main/views.py

```diff
@@ -17,11 +20,14 @@ def upload_circuit(request):
 
 def view_compiled(request):
     """ url shortcut name: lattice_main-latticeview """
-    # Save the file to a tmp dir so that pyzx can read it
-    print("post-request",request.FILES['circuit'].name)
 
+    #print("post-request",request.FILES['circuit'].name)
     file_ext = request.FILES['circuit'].name.split('.')[-1]
-    circuit_tmp_save_location = os.path.join('/tmp','%s.%s'%(uuid.uuid4(),file_ext))
+    # Save the file to a tmp dir so that pyzx can read it
+    ### for Linux deployment ###
+    #circuit_tmp_save_location = os.path.join('/tmp','%s.%s'%(uuid.uuid4(),file_ext))
+    ### Windows Testing ###
+    circuit_tmp_save_location = os.path.join('C:/Users/Keelan/Documents/Websites and Programming Projects/Lattice Surgery/temp server files','%s.%s'%(uuid.uuid4(),file_ext))
```

Comment:
We can probably find a way to not have to hard code paths :thinking:

Reply:
Yes, the above is outdated now. I created a folder internal to the project in /web/lattice_main/tmp which is used instead. I think I'm still using the relative to absolute path tool os.path.abspath as you used previously.